source/base/Text.ooc

```diff
@@ -53,6 +53,30 @@ Text: cover {
 	operator != (other: String) -> Bool { !(this == other) }
 	operator != (other: This) -> Bool { !(this == other) }
 	operator + (other: This) -> This { This new(this _buffer + other _buffer) }
+	operator + (other: Int) -> This {
+		converted := other toString()
```

Comment:
Needing to add `ooc-base` to call `toText` is not so surprising, since we're getting a type (`Text`) that is in base anyhow. But I see your point, I will change to use extensions instead.